extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -142,12 +142,12 @@ export class DeployConfigPage extends DacFxConfigPage {
 
 		//Handle database changes
 		this.databaseDropdown.onValueChanged(() => {
-			const databaseDropdownValue = this.databaseDropdown.value as azdata.CategoryValue;
+			const databaseDropdownValue = this.databaseDropdown.value;
```

Comment:
can this var be specified as a string instead of casting it later? that'd ensure compile-time correctness